android/BOINC/app/src/main/java/edu/berkeley/boinc/rpc/Notice.java

```diff
@@ -47,6 +47,10 @@ public int describeContents() {
         return 0;
     }
 
+    public double getCreate_time() {
```

Comment:
Looks like this function is created but never used.
What is the purpose of this function when it just returns a public variable?

Reply:
My apologies, I thought I had removed that change.

Reply:
Now your PR contains commits from master. Could you please make a clean rebase on latest master to remove all commits that are not related to this PR and merge commits too? Thanks in advance

Reply:
Yeah, of course.